app/utils/lava_log_parser.py

```diff
@@ -73,11 +76,35 @@ def run(log, boot, txt, html):
     else:
         boot_result_html = "<span class=\"warn\">{}</span>".format(boot_result)
 
+    # panic, error and warn are considered deprecated, but still here 
+    # for ensure that everything will works well in any case.
     formats = {
```

Comment:
We need a separate list for the kernel log levels.  You can then use the numeric log level to look up the colour in the list.

Reply:
yes, I saw that values doesn't ensure a sorted list. I was trying to thinking in solution to use same structure..